plankton/devices/linkam_t95/interfaces/stream_interface.py

```diff
@@ -21,6 +21,19 @@
 
 
 class LinkamT95StreamInterface(StreamAdapter):
+    """
+    Linkam T95 TCP stream interface
+
+    This is the interface of a simulated Linkam T95 device. The device listens on a configured
+    host:port-combination, one option to connect to it is via telnet:
+
+        $ telnet host port
+
+    Once connected, it's possible to send the specified commands, described in the dynamically
+    generated documentation. Information about host, port and line terminators in the concrete
+    device instance are also generated dynamically.
```

Comment:
Ah yes, here is the gist:

https://gist.github.com/MichaelWedel/fd288ac19ef1d08d994e639be78aadd0